clang/test/dpct/parse_build_log/compile_commands.json_ref

```diff
@@ -0,0 +1,12 @@
+// CHECK:[
+// CHECK:    {
+// CHECK:        "command": "nvcc -c -I../../common/inc -m64 --std=c++11 -o bar.o -D__CUDACC__=1 bar.cu",
+// CHECK:        "directory": "{{.*}}/parse_build_log/Output",
+// CHECK:        "file": "{{.*}}/parse_build_log/Output/bar.cu"
```

Comment:
Line 5,  the bar.cu is in parse_build_log folder, so the path here is wrong. 
Will the scripts file check if  " "{{.*}}/parse_build_log/Output/bar.cu" is exist to hint whether -work-dir is set right.

Reply:
Okay